src/OrbitClientModel/CaptureDeserializerLoadFuzzer.cpp

```diff
@@ -18,13 +19,14 @@
 #include "OrbitClientModel/CaptureDeserializer.h"
 #include "OrbitClientModel/CaptureSerializer.h"
 
+ABSL_FLAG(uint16_t, sampling_rate, 1000, "Frequency of callstack sampling in samples per second");
+ABSL_FLAG(bool, frame_pointer_unwinding, false, "Use frame pointers for unwinding");
```

Comment:
Same here: what are these flags for here and why only these?

Reply:
Do we need a TODO, or are you already working on that?

Reply:
#2103